### ISSUE TITLE
Display attributes of a component in the sidebar

### ISSUE THREAD

**Issue by rigor789** [vuejs/vue-devtools#734]
### What problem does this feature solve?
Right now there is no way to see all the attributes passed to a component. In some cases it could be useful to see the values of `$attrs` in the sidebar, especially for functional components. 

Additionally it would be nice to have an option to show the attributes in the component tree - this may get messy in some cases, so this would need to be opt-in via settings.

Reason for this feature request: Once #732 is merged we are one step closer to bringing VueDevtools into the NativeScript-Vue toolchain. In the upcoming 2.0 release of NativeScript-Vue we wrapped all elements into functional components, to be able to see them in VueDevtools, these elements can have attrivutes and it would help debug if we could see the values. 

Example: 
```xml
<ActionBar class="action-bar" title="Hello world"/>
```
(which is a component where we don't want to create props for every property) currently renders like so 
<img width="804" alt="screen shot 2018-08-05 at 5 44 17" src="https://user-images.githubusercontent.com/879060/43682345-b5d7664e-9872-11e8-9628-dd51e628cbec.png">

it would be nice to see `title: "Hello world"` and `class: "action-bar"` in the component inspector

If this is something that would be considered, I'm willing to submit a PR with the changes.

### What does the proposed API look like?
N/A

**Comment by sqal** [comment]
> especially for functional components

but functional components are not displayed in components panel 😕

**Comment by rigor789** [comment]
@sqal https://github.com/vuejs/vue-devtools/pull/719 :)
